test/test.event.js

```diff
@@ -27,7 +27,7 @@ describe("Event", function() {
 
   describe("#getEventInfo",function(){
     function testNumber(info,name,expected) {
-      it("should return the good value for " + name, function () {
+      skip().vendor("nVidia").it("should return the good value for " + name, function () {
```

Comment:
more skipped tests

Reply:
I see more need of incremental migration to TypeScript support.